core/src/elo/elo.resolver.ts

```diff
@@ -0,0 +1,25 @@
+import {Mutation, Resolver} from "@nestjs/graphql";
+
+import {EloService} from "./elo.service";
+import {EloConnectorService, EloEndpoint} from "./elo-connector";
+
+@Resolver()
+export class EloResolver {
+    constructor(
+        private readonly eloService: EloService,
+        private readonly eloConnectorService: EloConnectorService,
+    ) {}
+
+    @Mutation(() => String)
+    async generateMigrationData(): Promise<string> {
+        await this.eloService.prepMigrationData();
+        return "Done.";
+    }
+
+    @Mutation(() => String)
+    async runEloMigration(): Promise<string> {
+        const inputData = await this.eloService.prepMigrationData();
+        await this.eloConnectorService.createJob<EloEndpoint.AddNewPlayers>(EloEndpoint.AddNewPlayers, inputData);
```

Comment:
Should need generic type here; it comes from the enum parameter automatically. If not, that may be a fix on my branch.